test_autobee.js

```diff
@@ -0,0 +1,1074 @@
+const test = require('tape')
```

Comment:
Could you refactor `test.js` so that we can have all the tests take a `getBee` parameter which will be run with both the regular hyperbee and the autobee?

This way we don't need to duplicate all the test code and can make sure that new tests cover both sides.

e.g.

```JavaScript
makeTests(getBee, 'Hyperbee')
makeTests(getAutoBee, 'Autobee')

function makeTests(getBee, type) {
  test(`${type} Create a document in a collection`, async (t) => {
  const db = new DB(getBee())
.... etc
}